openwisp_monitoring/device/static/monitoring/css/daterangepicker.css

```
@@ -32,7 +32,7 @@
 .daterangepicker:after {
   top: -6px;
   border-right: 6px solid transparent;
-  border-bottom: 6px solid #fff;
+  border-bottom: 6px solidvar(--ow-color-white);
```

Biome (2.4.6)

[error] Unexpected unknown function: solidvar

(lint/correctness/noUnknownFunction)